qa/Rakefile

```diff
@@ -0,0 +1,66 @@
+require 'rspec'
+require 'rspec/core/rake_task'
+
+task :spec    => 'spec:all'
+task :default => :spec
+
+namespace :test do
+
+  def fetch_config
+    require "open3"
+    Open3.popen3("vagrant ssh-config") do |stdin, stdout, stderr, wait_thr|
+      { :stdout => stdout.read.chomp, :stderr => stderr.read.chomp,
+        :exit_status => wait_thr.value.exitstatus }
+    end
+  end
+
+  def parse(lines)
+    hosts, host = [], {}
+    lines.each do |line|
+      if line.match(/Host\s(.*)$/)
+        host = { :host => line.gsub("Host","").strip }
+      elsif line.match(/HostName\s(.*)$/)
+        host[:hostname] = line.gsub("HostName","").strip
+      elsif line.match(/Port\s(.*)$/)
+        host[:port]     = line.gsub("Port","").strip
+      elsif line.empty?
+        hosts << host
+        host = {}
+      end
+    end
+    hosts << host
+    hosts
+  end
+
+  desc "generate a valid ssh-config"
+  task :ssh_config do
+    require "json"
+    cd "acceptance" do
+      File.write("../.ssh_config", parse(fetch_config[:stdout].split("\n")).to_json)
+    end
+  end
```

Comment:
instead of your `.vagrant_ssh_config`, I will be using `.vm_ssh_config` as is less tool dependant and more descriptive as it relates to the vm's used for testing.